types/utils.go

```diff
@@ -29,3 +33,22 @@ func MustSortJSON(toSortJSON []byte) []byte {
 	}
 	return js
 }
+
+// DefaultChainID returns the chain ID from the genesis file if present. An
+// error is returned if the file cannot be read or parsed.
+//
+// TODO: This should be removed and the chainID should always be provided by
+// the end user.
+func DefaultChainID() (string, error) {
```

Comment:
Mhmmm don't think this is the best place for this (client/utils would prob be better), but I don't want to block this PR.